### PR TITLE
feat: add home route + autocomplete search widget

### PR DESCRIPTION
**Pull request by bertyhell**
closes: https://district01.atlassian.net/browse/AVO2-259

image

### CODE REVIEW

File: src/home/views/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+			0,
+			ITEMS_IN_AUTOCOMPLETE,
+			{
+				query: debouncedSearchTerms || '',
+			},
+			{}
+		);
+	}, [debouncedSearchTerms]);
+
+	const gotoSearchResult = (searchResultId: string | number | undefined) => {
```
Comment (benjaminnaesen) [inline src/home/views/Home.tsx]: `goToSearchResult` :grammar_nazi:

File: src/home/views/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+const ITEMS_IN_AUTOCOMPLETE = 5;
+
+const Home: FunctionComponent<HomeProps> = ({
+	searchResults,
+	searchResultsLoading,
+	search,
+	history,
+}: HomeProps) => {
```
Comment (andrych) [inline src/home/views/Home.tsx]: `: HomeProps` mag hier weg

File: src/shared/helpers/useDebounce.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+	useEffect(() => {
+		const handler = setTimeout(() => {
+			setDebouncedValue(value);
+		}, delay);
+
+		return () => {
+			clearTimeout(handler);
+		};
+	}, [value]);
```
Comment (e2o) [inline src/shared/helpers/useDebounce.ts]: delay ontbreekt hier in uw deps?

File: src/home/views/Home.tsx [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+	Container,
+	Dropdown,
+	DropdownButton,
+	DropdownContent,
+	MenuSearchResultContent,
+	Spacer,
+	Spinner,
+	TextInput,
+} from '@viaa/avo2-components';
+import { MenuSearchResultItemInfo } from '@viaa/avo2-components/dist/components/MenuSearchResult/MenuSearchResultContent';
```
Comment (e2o) [inline src/home/views/Home.tsx]: Fix export in barrel van de components?

**Pull request merged**